php/5.6/node/8/scripts/docker-as-drupal

```diff
@@ -34,6 +34,7 @@ BROWSERTEST_OUTPUT_DIRECTORY="/var/www/web/sites/simpletest"
 DATABASE_HOST=$(a=${DATABASE_URL#*@}; echo ${a%/*})
 DATABASE_NAME=${DATABASE_URL##*/}
 DATABASE_USER=$(a=${DATABASE_URL#*//}; echo ${a%:*})
+DATABASE_PASSWORD=$(a=${DATABASE_URL#*//}; b=${a#*:}; echo ${b%@*})
```

Comment:
I've no idea what `b` is assigned to, but I trust you :)

Reply:
it's *easy" 😂 

We have "mysql://user:password@host/database" in DATABASE_URL. First set `a` with eversthing after `//`, so `a = "user:password@host/database"`, then set `b` with evetything after `:`, so `b = "password@host/database"`, and to finish return everything in `b` before `@ ` to have only the password.